client.toml

```diff
@@ -0,0 +1,16 @@
+[Logging]
+  Disable = false
+  Level = "DEBUG"
+  File = ""
+
+[UpstreamProxy]
+  Type = "none"
+
+[Debug]
+  DisableDecoyTraffic = true
```

Comment:
Add a disclaimer that for non-debug purposes, `DisableDecoyTraffic` should be set to `false`. Clients should be periodically sending decoy traffic to providers.